shared/js/background/tab-manager.es6.js

```diff
@@ -64,6 +64,27 @@ class TabManager {
         settings.updateSetting(list, globalwhitelist)
     }
 
+    /* Update icon and popup of every open tab for opt in experiment
+     */
+    handleTrackerBlockingToggle (value) {
+        for (let tabId in this.tabContainer) {
+            let tab = this.tabContainer[tabId]
+            if (tab.site) {
+                if (value) {
+                    tab.site.setWhitelistStatusFromGlobal()
+                    tab.site.blockingDisabled = false
+                    // update post install page when tracker blocking enabled in popup
+                    if (tab.url.match(/duckduckgo\.com\/app\?post=1/)) {
+                        chrome.tabs.sendMessage(parseInt(tabId, 10), {trackerBlockingEnabled: true, url: tab.url})
```

Comment:
If a user hits the post install page and opens the popup, then activates tracker blocking from there, the page reloads. We need the page to remember that they already activated tracker blocking and show the success state.

To do this, we pass a message to the content script injected into the post install page, which then passes the message on to the actual page code. When the page gets this message, it sets a cookie to remember that the user already activated tracker blocking. This way, when the post install page reloads it loads the success state.